commands_config/config_command.go

```
@@ -91,9 +92,18 @@ func setupValueWithKeyInputInvitation(configKey string, recommendationMsg string
 	}
 	newValue := scanner.Text()
 	if newValue != "" {
+
+		println("Config key is: " + configKey)
+		println("App config name is: " + appConfigName)
+		oldConfig, _ := utils.LoadConfig(configFilePath)
+		utils.PrintFieldValue(oldConfig, appConfigName, "OLD")
 		viper.SetConfigFile(configFilePath)
 		viper.ReadInConfig()
 		viper.Set(configKey, newValue)
 		viper.WriteConfig()
+		newConfig, _ := utils.LoadConfig(configFilePath)
```

do not ignore error :)